automated install/basic-install.sh

```diff
@@ -1923,6 +1917,8 @@ installPihole() {
                 printf "  %b Warning: 'lighty-enable-mod' utility not found\\n" "${INFO}"
                 printf "      Please ensure fastcgi is enabled if you experience issues\\n"
             fi
+            # Restart lighttpd to make the new group membership effective
+            restart_service "lighttpd"
```

Comment:
Lighttpd should already be restarted here, after `installPihole` is run:
https://github.com/pi-hole/pi-hole/blob/bc8976fe755a9f7eebbd63847db68e869389bdb4/automated%20install/basic-install.sh#L2636-L2645

Reply:
Thank you for pointing this out, I'll remove this commit from this PR.

The user reporting the issue mentioned meanwhile that they use `apache2` and have `INSTALL_WEB_SERVER == false` which is a configuration we do not directly support.